### PR TITLE
docs: Pretty print install.md

### PR DESCRIPTION
**Pull request by iiiceoo**
Signed-off-by: iiiceoo <ziqian.xue@daocloud.io>

### CODE REVIEW

File: docs/usage/install.md [hunk 1]
```diff
@@ -1,167 +1,186 @@
...
+# Installation
 
-The spiderpool needs to install webhook of kube-apiserver, so the TLS certificates are required.
+*This guide shows how to install Spiderpool using [Helm](https://helm.sh/).* 
 
-There are two ways to install it, the one is with cert-manager, the other one is to generate self-signed certificates.
+## Generic
 
-## Install spiderpool
+Setup the Helm repository.
```
Comment (windsonsea) [inline docs/usage/install.md:7]:
```suggestion
Set up the Helm repository.
```

File: docs/usage/install.md [hunk 2]
```diff
@@ -1,167 +1,186 @@
...
-The following is a dual-stack example.
+## Certificates
 
-```shell
-helm repo add spiderpool https://spidernet-io.github.io/spiderpool
+Spiderpool-controller needs TLS certificates to run webhook server. We can configure it in several ways.
+
+### Auto
+
+Use Helm's template function [genSignedCert](https://helm.sh/docs/chart_template_guide/function_list/#gensignedcert) to generate TLS certificates, this is the simplest and most common way to configure:
```
Comment (windsonsea) [inline docs/usage/install.md:45]:
```suggestion
Use Helm's template function [genSignedCert](https://helm.sh/docs/chart_template_guide/function_list/#gensignedcert) to generate TLS certificates. This is the simplest and most common way to configure:
```

File: docs/usage/install.md [hunk 3]
```diff
@@ -1,167 +1,186 @@
...
+```bash
+helm install spiderpool spiderpool/spiderpool --namespace kube-system \
+  --set spiderpoolController.tls.method=auto
+```
+
+Note that the default value of parameter `spiderpoolController.tls.method` is `auto`.
+
+### Provided
+
+If you want to run spiderpool-controller with a self-signed certificate, `provided` would be a good choice. You can use OpenSSL to generate certificates, or our script:
```
Comment (windsonsea) [inline docs/usage/install.md:56]:
```suggestion
If you want to run spiderpool-controller with a self-signed certificate, `provided` would be a good choice. You can use OpenSSL to generate certificates, or run the following script:
```

File: docs/usage/install.md [hunk 4]
```diff
@@ -1,167 +1,186 @@
...
+
+```bash
+# "CIDR"
+IPV4_SUBNET_YOU_EXPECT="172.18.40.0/24"
+# "IP" or "IP-IP"
+IPV4_IPRANGES_YOU_EXPECT="172.18.40.40-172.20.40.200"
 ```
 
-### Install by cert-manager
+Let's create an IPv4 IPPool while deploying Spiderpool, and configure it as the cluster default IPPool in the [global Configmap configuration](https://github.com/spidernet-io/spiderpool/blob/main/docs/concepts/config.md#configmap-configuration).
```
Comment (windsonsea) [inline docs/usage/install.md:107]: How about a relative url? such as `../concepts/config.md#configmap-configuration`

File: docs/usage/install.md [hunk 5]
```diff
@@ -1,167 +1,186 @@
...
 
-After installation of the spiderpool, please edit the CNI configuration file under `/etc/cni/net.d/`.
+```json
+"ipam":{
+    "type":"spiderpool"
+}
+```
 
-The following is an example for macvlan CNI.
+Take a example of Macvlan + Spiderpool:
```
Comment (windsonsea) [inline docs/usage/install.md:158]:
```suggestion
Take an example of Macvlan + Spiderpool:
```

**Review by windsonsea — COMMENTED** [review]
Some suggestions about text.

File: docs/usage/install.md [hunk 6]
```diff
@@ -1,167 +1,186 @@
...
 }
 ```
 
-Refer to [config](../concepts/config.md) for details about the IPAM configuration.
+## Uninstall
+
+Generally, we can uninstall Spiderpool release in this way:
+
+```bash
+helm uninstall spiderpool -n kube-system
```
Comment (weizhoublue) [inline docs/usage/install.md:178]: 可能 还有残留

**Review by weizhoublue — APPROVED** [review]

**Review by Icarus9913 — APPROVED** [review]

**Pull request merged**